Shared/alerts/DynamicEntityAlertSource.cpp

```diff
@@ -87,10 +89,14 @@ QVariant DynamicEntityAlertSource::value(const QString& key) const
  */
 void DynamicEntityAlertSource::setSelected(bool selected)
 {
+  // check to ensure that the dynamic entity has not been released
+  if (!m_dynamicEntity)
```

Comment:
This was a default guard in the event that the MessageFeed purged the DynamicEntity but this AlertCondition for it still exists. I don't think there is any mechanism in the AlertConditionsController that will remove them when an entity is purged. This may not have been an issue when Graphics were used.